build/Common.props

```diff
@@ -45,8 +45,8 @@
     <SystemCollectionsImmutablePkgVer>1.4.0</SystemCollectionsImmutablePkgVer>
     <SystemDiagnosticSourcePkgVer>7.0.0-rc.2.22472.3</SystemDiagnosticSourcePkgVer>
     <SystemReflectionEmitLightweightPkgVer>4.7.0</SystemReflectionEmitLightweightPkgVer>
-    <SystemTextJsonPkgVer>4.7.0</SystemTextJsonPkgVer>
-    <SystemThreadingTasksExtensionsPkgVer>4.5.3</SystemThreadingTasksExtensionsPkgVer>
+    <SystemTextJsonPkgVer>4.7.2</SystemTextJsonPkgVer>
+    <SystemThreadingTasksExtensionsPkgVer>4.5.4</SystemThreadingTasksExtensionsPkgVer>
```

Comment:
This is also updated due to "/home/runner/work/opentelemetry-dotnet/opentelemetry-dotnet/src/OpenTelemetry.Exporter.Jaeger/OpenTelemetry.Exporter.Jaeger.csproj : error NU1605:  OpenTelemetry.Exporter.Jaeger -> System.Text.Json 4.7.2 -> System.Threading.Tasks.Extensions (>= 4.5.4)  [/home/runner/work/opentelemetry-dotnet/opentelemetry-dotnet/OpenTelemetry.sln]"